apps/portals-base/src/components/BrowseToolsPage.tsx

```diff
@@ -43,7 +25,13 @@ const categories: Category[] = [
   { resourceName: 'Biobank', image: <Biobanks /> },
 ]
 
-const BrowseToolsPage = () => {
+export type BrowseToolsPageProps = {
+  popularSearchesSql: string
+  toolsSql: string
+}
```

Comment:
This 'shared' component imported code directly from the NF config. Since the NF config is now in a separate project, I did the bare minimum to convert the config data into props.